Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -2560,9 +2560,9 @@ extension BrowserViewController {
         user.referrals.pendingClaim != nil
     }
 
-    func presentInsightfulSheetsIfNeeded() -> Bool {
+    func presentInsightfulSheetsIfNeeded() {
```

Comment:
I guess an Ecosia comment here should also be added?

Reply:
This is part of the old implementation and was indeed missing an Ecosia supporting comment 👍 , will do it as part of this changes 💪